src/admin/views/BannersView.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {Helmet} from 'react-helmet';
+import {useHistory} from 'react-router-dom';
+import {useQuery, useMutation} from '@apollo/react-hooks';
```

Comment:
You want to update this to import from @apollo/client - `import {useQuery, useMutation} from '@apollo/client';`